content-scripts/src/modules/initialize.js

```diff
@@ -20,3 +21,223 @@ export const injectOverrideFunctions = () => {
   script.src = chrome.runtime.getURL("js/override-functions.js");
   document.body.appendChild(script);
 }
+
+/**
+ * Reverse the date direction
+ * Ex: 2023-10-05 to 05-10-2023
+ */
+export const reverseDateDirection = () => {
+  document.querySelectorAll(".data").forEach(date => {
+      date.innerHTML = date.innerHTML.split('-').reverse().join('-');
```

Comment:
This may be a little dangerous, we should probably check if the content is an actual date firts

Reply:
Agree
What you think about convert the string to Date and verify if it creates a valid Date object?
`date instanceof Date && !isNaN(date)`

Reply:
Sounds like a plan